roles/st2chatops/tasks/main.yml

```diff
@@ -26,18 +27,55 @@
   when: st2chatops_version != "latest"
   tags: [st2chatops, skip_ansible_lint]
 
-- name: Add st2_api_key in st2chatops.env
+- name: Check if api key already exist. No change if it already exists
+  become: yes
+  shell: "cat /opt/stackstorm/chatops/st2chatops.env | grep -e '^export ST2_API_KEY=\"\\${ST2_API_KEY}\"$'"
```

Comment:
@arma: `${ST2_API_KEY}` from here: https://github.com/StackStorm/st2chatops/blob/master/st2chatops.env#L29 to take care of: `Don't overwrite st2chatops.env if there is existing api key`.